src/services/git/GitServiceUtils.ts

```diff
@@ -1,16 +1,19 @@
 import gitUrlParse from 'git-url-parse';
-import { GitService } from './model';
+import _ from 'lodash';
+import { ServiceType } from '../../detectors/IScanningStrategy';
 import { assertNever } from '../../lib/assertNever';
+import { ProjectComponent } from '../../model';
+import { RepositoryConfig } from '../../scanner/RepositoryConfig';
 
 export class GitServiceUtils {
   static getUrlToRepo = (url: string, path?: string | undefined, branch = 'master') => {
     const parsedUrl = gitUrlParse(url);
 
     let completeUrl = `${parsedUrl.protocol}://${parsedUrl.resource}/${parsedUrl.owner}/${parsedUrl.name}`;
-    const sourceUrl = <GitService | null>parsedUrl.resource;
+    const sourceUrl = <ServiceType | null>parsedUrl.resource;
```

Comment:
⚠️ Is it really `ServiceType` interface in `parsedUrl.resource`? I guess there are differences in the letter casing.